static/src/javascripts/projects/common/modules/commercial/article-body-adverts.js

```diff
@@ -66,12 +66,10 @@ define([
         });
     }
 
-    function isMtRecTest() {
-        var MtRec1Test = ab.getParticipations().MtRec1,
-            MtRec2Test = ab.getParticipations().MtRec2;
+    function isViewabilityTest() {
```

Comment:
Yeah I rushed through it and forgot. I am replacing it with `ab.shouldruntest` so it will be much cleaner.